pkg/token/cache.go

```diff
@@ -132,3 +140,50 @@ func (c *LockedTokenCache) Clear() {
 	}
 	c.memoryUsage = 0
 }
+
+var (
+	tokenExpiresInMetric = "token_expires_in_seconds"
+	tokenExpiresInHelp   = "Indicates remaining time until the token expires."
+)
+
+func (c *LockedTokenCache) Describe(ch chan<- *prometheus.Desc) {
+	labelKeys := []string{"domain", "role"}
+	ch <- prometheus.NewDesc(tokenExpiresInMetric, tokenExpiresInHelp, labelKeys, prometheus.Labels{
+		"type": c.tokenType,
+	})
+}
+
+func (c *LockedTokenCache) Collect(ch chan<- prometheus.Metric) {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	for k, t := range c.cache {
```

Comment:
```suggestion
	for k, t := range c.cache {
	
	// skip placeholder token added during daemon creation
	// if t.domain = "" { continue }
```
https://github.com/AthenZ/k8s-athenz-sia/blob/6ac184b46db85a5e5fed275ddc3976b1320bc227/pkg/token/daemon.go#L73-L78

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/77/commits/732ace5f79e2f591ea5d7a96bd43c702a934453c